applications/UMC-Tokenscribe.md

```diff
@@ -0,0 +1,159 @@
+# Open Grant Proposal
+
+* **Project Name:** UMC - Tokenscribe
+* **Team Name:** UMC Labs
+* **Payment Address:** 1KoTuo8c2YomLXPqLdrPVEiufSDyCwrNGC
+
+## Project Overview :page_facing_up: 
+
+### Overview
+
+#### Introduction
+
+Subscription is a common payment method existed for a long time since credit cards. In the blockchain age, DeFi is the edge of web3 tech to change the world. In the meanwhile, there is no secured subscription feature for users to subscribe to their memberships nor their virtual staffs on-chain.
+As we know, the draft [EIP1337](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1337.md)  proposed a way to implement token subscription for [ERC20](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md)  tokens. But from the view of users, EIP1337 is not the secure way to implement token subscription.
+This application is aimed to propose a specification of secured token subscription which is the key feature of UMC - Universal Membership Community.
+
+#### Integration
+
+The secured token subscription will be implemented with Ink!.
```

Comment:
Have you done any development so far with ink! ?

Reply:
@lamafab  Yes, we have tried Ink!. We are trying to implement another project which will include contracts for token subscription to handle membership, and I realized that current EIP1337 and other solutions have potential security issue which will cause losing control of their own assets for users. So I propose a new way to implement token subscription as this application.

Reply:
Yes, we have tried Ink!. We are trying to implement another project which will include contracts for token subscription to handle membership, and I realized that current EIP1337 and other solutions have potential security issue which will cause losing control of their own assets for users. So I propose a new way to implement token subscription as this application.